jerry-libc/jerry-libc.c

```diff
@@ -62,9 +63,9 @@ memset (void *s,  /**< area to set values in */
         size_t n) /**< area size */
 {
   uint8_t *area_p = (uint8_t *) s;
-  for (size_t index = 0; index < n; index++)
+  while (n--)
   {
-    area_p[ index ] = (uint8_t) c;
+    *area_p++ = (uint8_t) c;
   }
```

Comment:
Same as above. Why introduce variables if you can do well without them? This is C, so `*p++` is not that much unheard of.